### PR TITLE
fixed broken link to issues page in contribution guide

### PR DESCRIPTION
**Pull request by opeolluwa**

### CODE REVIEW

**Comment by loftwah** [comment]
You are fantastic. Thank you for getting involved. I'll sort this side of the project out soon so issues and PRs can be adequately managed with GitHub Actions. 🙇‍♂️🌟

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+    "workbench.colorCustomizations": {}
+}
```
Comment (loftwah) [inline .vscode/settings.json]: Can I please delete this file?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ To contribute to the Linux for Pirates! guide, please follow these steps:
 
 > **Note** No pull requests will be accepted without an issue, except in specific cases that will be approved on an ad-hoc basis.
 
-- To open an issue, go to the [issue tracker](https://github.com/YOUR-USERNAME/linux-for-pirates/issues) and click the "New Issue" button. Provide a clear and concise description of your proposed contribution, and any relevant details or questions.
+- To open an issue, go to the [issue tracker](https://github.com/loftwah/linux-for-pirates/issues) and click the "New Issue" button. Provide a clear and concise description of your proposed contribution, and any relevant details or questions.
```
Comment (loftwah) [inline CONTRIBUTING.md:49]: Ahh, got you. Great pickup! I must have copy-pasted it in the wrong spot. I didn't mean to use that URL for the issues part. Thanks!

Comment (opeolluwa) [reply CONTRIBUTING.md:49]: Oh! Alright

**Comment by opeolluwa** [comment]
> You are fantastic. Thank you for getting involved. I'll sort this side of the project out soon so issues and PRs can be adequately managed with GitHub Actions. 🙇‍♂️🌟

That would be awesome. Cheers

File: .vscode/settings.json (hunk 1 above)
Comment (opeolluwa) [reply .vscode/settings.json]: > Can I please delete this file?

Yes

**Review by loftwah — APPROVED** [review]

**Pull request merged**